src/pages/partners/payment-agent/_premium_payment_agent.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+    wrap_container,
+    array_wrapper,
+    heading,
+    country,
+    box,
+    heading_countries,
+} from '../_premium_agent.module.scss'
+import { TString } from 'types/generics'
+import Container from 'features/components/atoms/container'
+import { Localize } from 'components/localization'
+import Image from 'features/components/atoms/image'
+import Typography from 'features/components/atoms/typography'
+import FlexBox from 'features/components/atoms/flex-box'
+import useScreenSize from 'components/hooks/use-screen-size'
+import Botswana from 'images/svg/flags/botswana.svg'
+import Brazil from 'images/svg/flags/brazil.svg'
+import India from 'images/svg/flags/india.svg'
+import Kenya from 'images/svg/flags/kenya.svg'
+import Nigeria from 'images/svg/flags/nigeria.svg'
+import SouthAfrica from 'images/svg/flags/south-africa.svg'
+import SriLanka from 'images/svg/flags/sri-lanka.svg'
+import Tanzania from 'images/svg/flags/tanzania.svg'
+import Zimbabwe from 'images/svg/flags/zimbabwe.svg'
+
+interface CountryProps {
```

Comment:
```suggestion
type TCountry = {
```

Reply:
Hi @NikitK-deriv As we discussed in our session we won’t go with prefix of `T` and `I` for type and interface and we will refactor the ones we have 🙏 

cc: @mohsen-deriv

Reply:
Clear, thanks @mitra-deriv